README.md

```diff
@@ -12,10 +12,11 @@ We assume that you know how to use Bevy already and have seen the [official Quic
 
 ## Create a new game
 
-Install [`cargo-generate`](https://github.com/cargo-generate/cargo-generate) and run the following command:
+Install [`cargo-generate`](https://github.com/cargo-generate/cargo-generate) and run the following commands:
 
 ```sh
 cargo generate TheBevyFlock/bevy_quickstart --branch cargo-generate
+git switch --create main
```

Comment:
This is fine as is. But curious if this could / should be a command to rename the current branch from `cargo-generate` to `main`?

Reply:
I would personally prefer to run a single simple `git` command than a mysterious blackbox.

Reply:
Oh, I think this would just be `git branch --move main` according to stackoverflow.

Reply:
lemme check

Reply:
You're right, TIL. [`940a909` (#161)](https://github.com/TheBevyFlock/bevy_quickstart/pull/161/commits/940a909c6210692b6ebb3f0eac49ecb0700ca800)